tests/ui/consts/min_const_fn/const-fn-lang-feature.rs

```diff
@@ -0,0 +1,19 @@
+//! Ensure that we can use a language feature with a `const fn` and things behave as expected.
```

Comment:
Explain what "expected" means in detail.

Reply:
Well, it should work like all feature gates work -- if you enable the feature gate it is actually considered enabled...